### ISSUE TITLE
filed.type throws unknown method error

### ISSUE THREAD

**Issue by SanboKyo** [verbb/formie#424]
All field attributes (id, name, label, handle, ...) works except for type.

That throws: Calling unknown method: verbb\formie\fields\formfields\SingleLineText::type()

(Same with other fields.)

**Additional info**
- Plugin version: dev-craft-3 as 1.3.26
- Craft version: 3.11.2
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.26",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.27](https://github.com/verbb/formie/releases/tag/1.3.27)
